src/startupscriptgenerator/src/dotnetcore/scriptgenerator.go

```diff
@@ -42,11 +42,14 @@ func (gen *DotnetCoreStartupScriptGenerator) GenerateEntrypointScript(scriptBuil
 	}
 
 	dotnetBinary := "/usr/share/dotnet/dotnet"
+
 	enableDynamicInstall := common.GetBooleanEnvironmentVariable(consts.EnableDynamicInstallKey)
 	if enableDynamicInstall && !common.PathExists(dotnetBinary) {
 		scriptBuilder.WriteString(fmt.Sprintf("oryx setupEnv -appPath %s\n", appPath))
 	}
 
+	common.SetupPreRunScript(scriptBuilder)
```

Comment:
`&scriptBuilder`

Reply:
The scriptBuilder within dotnet script generator is already a pointer type.

Reply:
Gotcha thanks